parsers/MX.py

```diff
@@ -228,6 +264,44 @@ def fetch_exchange(
     return data
 
 
+@refetch_frequency(timedelta(hours=1))
+def fetch_consumption(
+    zone_key: ZoneKey,
+    session: Optional[Session] = None,
+    target_datetime: Optional[datetime] = None,
+    logger: Logger = getLogger(__name__),
+) -> list:
+    """Gets the consumption data for a region using the live dashboard."""
+    # TODO the calls could be improved since we can get all the data in one call.
+    if session is None:
+        session = Session()
+    if target_datetime is not None:
+        raise NotImplementedError("This parser is not yet able to parse past dates")
+    response = session.get(MX_EXCHANGE_URL)
+    if response.status_code != 200:
```

Comment:
should we use response.ok here instead? 
```suggestion
    if not response.ok:
```